File: testprojects/csharp/benchmark/infrastructure/socialmedia/pinterest/PinDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CSharpBenchmark.infrastructure.socialmedia.pinterest
{

    public class PinDAO
    {

        public BoardDAO getBoardFromAPI()
        {
            return new BoardDAO();
        }

        public bool login(PinterestUserDTO dto)
        {
            return false;
        }

        public BoardDTO getBoardDetails(Object o)
        {
            return new BoardDTO();
        }
    }
}

File: testresources/java/benchmark_2012/infrastructure/socialmedia/pinterest/PinDAO.java
package infrastructure.socialmedia.pinterest;

public class PinDAO {

	public BoardDAO getBoardFromAPI(){
		return new BoardDAO();
	}

	public boolean login(PinterestUserDTO dto) {
		return false;
	}
	
	public BoardDTO getBoardDetails(Object o){
		return new BoardDTO();
	}
}